rocketpool/node/collectors/node-collector.go

```diff
@@ -339,6 +352,33 @@ func (collector *NodeCollector) Collect(channel chan<- prometheus.Metric) {
 		return nil
 	})
 
+	// get the beacon client sync status:
+	wg.Go(func() error {
+		progress := float64(0)
+
+		syncStatus, err := collector.bc.GetSyncStatus()
+
+		if err != nil {
+			// don't return an error because then the metric isn't emitted at all, just let the Progress be 0
+			fmt.Printf("error getting beacon chain sync status: %w", err)
```

Comment:
revisiting this i think `return nil` here (after printing the error) would be good:

On startup, we don't report sync progress _until_ the beacon client is up.
After we get sync progress at least once, the last know sync progress will continue to be reported.

This way, the endpoint stays response, but we also use slightly more accurate signals